### PR TITLE
Add a users section to the admin area

### PR DESCRIPTION
**Pull request by adrianbanks**

### CODE REVIEW

File: DDDEastAnglia/Areas/Admin/Content/admin.css [hunk 1]
```diff
@@ -1,4 +1,8 @@
-﻿.brand a {
+﻿a { text-decoration: underline; }
+
+a.btn, .dropdown-menu a { text-decoration: none; }
```
Comment (alastairs) [inline DDDEastAnglia/Areas/Admin/Content/admin.css]: Could combine this with the next.

File: DDDEastAnglia/Areas/Admin/Models/UserModel.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace DDDEastAnglia.Areas.Admin.Models
+{
+    public class UserModel
```
Comment (alastairs) [inline DDDEastAnglia/Areas/Admin/Models/UserModel.cs:6]: Presumably this is a View Model and not a Data Model?

File: DDDEastAnglia/Areas/Admin/Models/UserModel.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        public int UserId { get; set; }
+        
+        [DisplayName("username")]
```
Comment (alastairs) [inline DDDEastAnglia/Areas/Admin/Models/UserModel.cs:10]: Why aren't these Title Case or Sentence case?

File: DDDEastAnglia/Areas/Admin/Content/admin.css (hunk 1 above)
Comment (adrianbanks) [inline DDDEastAnglia/Areas/Admin/Content/admin.css]: True. Will update.

File: DDDEastAnglia/Areas/Admin/Models/UserModel.cs (hunk 2 above)
Comment (adrianbanks) [inline DDDEastAnglia/Areas/Admin/Models/UserModel.cs:6]: Yes.

File: DDDEastAnglia/Areas/Admin/Models/UserModel.cs (hunk 3 above)
Comment (adrianbanks) [inline DDDEastAnglia/Areas/Admin/Models/UserModel.cs:10]: They get styled in sentence case in the view. Doing it this ways makes the validation messages read in the correct case.

File: DDDEastAnglia/Areas/Admin/Views/User/Index.cshtml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+<h2>@ViewBag.Title</h2>
+
+<div class="pull-right">
+    <strong>@Model.Count() users</strong>
+</div>
+
+<table id="usersTable" class="table table-striped table-hover table-condensed">
+    <thead>
+        <tr><th>
```
Comment (alastairs) [inline DDDEastAnglia/Areas/Admin/Views/User/Index.cshtml:20]: Slightly weird formatting, but ok.

File: DDDEastAnglia/Areas/Admin/Models/UserModel.cs (hunk 3 above)
Comment (alastairs) [inline DDDEastAnglia/Areas/Admin/Models/UserModel.cs:10]: Funky. Ok.

**Comment by alastairs** [comment]
Good to merge once that small CSS change is done.

**Comment by adrianbanks** [comment]
Updated.

File: DDDEastAnglia/Areas/Admin/Models/UserModel.cs (hunk 2 above)
Comment (alastairs) [inline DDDEastAnglia/Areas/Admin/Models/UserModel.cs:6]: I'll merge this now, but something that occurred to me is this doesn't appear to be a model for a specific View, and so isn't really a View Model. We should address that at some point.

**Pull request merged**